route-rs-runtime/src/processor/mod.rs

```diff
@@ -19,6 +19,12 @@ pub use self::drop::*;
 mod dec_ip_hop;
 pub use self::dec_ip_hop::*;
 
+mod log;
+pub use self::log::*;
```

Comment:
Nit: In the future, let's not do this with a wildcard, we should be only exporting the parts that are the interface.  In this case, the only struct there is an interface, so this is the same anyways. Up to you if you want to rectify.

Reply:
We'll probably go back and fix them all in one sweep, but I'll keep that in mind for next time.